packages/client/src/clients/in-app-message/channel-client.ts

```diff
@@ -0,0 +1,18 @@
+import Knock from "../../knock";
+
+import { InAppStore, createStore } from "./store";
```

Comment:
Nit, feel free to ignore - but I feel like these can just be brought into internal types/funcs or inlined into this module.

Reply:
I'm planning on adding a few more methods to make writing selectors easier, but if I don't end up adding much more, then I agree it'd be good to consolidate the files.